docs/src/main/sphinx/connector/hive.rst

```diff
@@ -398,6 +402,10 @@ Property Name                                      Description
                                                    Set to ``false`` to disable statistics. Disabling statistics
                                                    means that :doc:`/optimizer/cost-based-optimizations` can
                                                    not make smart decisions about the query plan.
+
+``hive.single-statement-writes``                   Each individual DML statement is required to be executed      ``false``
```

Comment:
What does this setting do...

Enable auto-commit mode so that each individual DML statement... 

or something like that.. also avoid using brackets. Just use a normal sentence.

Reply:
Changed the description to a more concise form.